samples/iam_block_public.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     - "organization/*"
     exclude: [] # optional, default is no exclusions
   parameters:
-    mode: blacklist
+    mode: denylist
```

Comment:
I'm a fan of succinctness, any thoughts on `deny` over having the type in the name?

Reply:
We already use [denylist](https://github.com/forseti-security/policy-library/search?q=denylist&unscoped_q=denylist) on quite a few templates, so I'm inclined to keep that as the standard unless you think there's a strong reason to change.

Reply:
No strong reason.  If allowlist/denylist is already the def-facto standard let's keep using that.